contextproject-ms3/src/main/java/contextproject/controllers/CLIController.java

```diff
@@ -0,0 +1,97 @@
+package contextproject.controllers;
+
+import contextproject.loaders.FolderLoader;
+import contextproject.models.Playlist;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Controls CLI for time being.
+ * 
+ * @author Felix
+ */
+public class CLIController {
+
+  private String fileName;
+  private Playlist result;
+
+  /**
+   * Input is initialised to empty string and result to null.
+   */
+  public CLIController() {
+    fileName = "";
+    result = null;
+
+  }
+  /**
+   * This will start a simple Command Line Interface for our first working version. Optionally to be
+   * included in later versions.
+   */
+  public void run() {
+    int input = 0;
+    Scanner sc = new Scanner(System.in);
+    System.out.println("Welcome to our application!");
+    while (input != 4) {
+      System.out.println("Option 1: Construct playlist! Enter foldername");
+      System.out.println("Option 2: About");
+      System.out.println("Option 3: Play music!");
+      System.out.println("Option 4: Close");
+      input = sc.nextInt();
+
+      switch (input) {
+        case 1 :
+          try {
+            this.setFileName(sc.nextLine());
+            FolderLoader folderLoader = new FolderLoader(this.getFileName());
```

Comment:
Why are we storing the fileName in the controller? We don't need it anywhere except for this method call.